hw/top_earlgrey/data/ip/chip_aes_testplan.hjson

```diff
@@ -0,0 +1,259 @@
+        "AES.MODE.CBC",
+        "AES.MODE.CFB-128",
+        "AES.MODE.OFB",
+        "AES.MODE.CTR",
+      ]
+      stage: V2
+      si_stage: SV3
+      lc_states: ["PROD"]
+      tests: []
+    }
+    {
+      name: chip_sw_aes_entropy
+      desc: '''Verify the AES entropy input used by the internal PRNGs.
+
+            - Write the initial key share, IV and data in CSRs (known combinations).
+            - Configure the entropy_src to generate entropy in LFSR mode.
+            - Write the PRNG_RESEED bit to reseed the internal state of the PRNG.
+            - Poll the status idle bit to ensure reseed operation is complete.
+            - Trigger the AES operation to run and wait for it to complete.
+            - Check the digest against the expected value.
+            - Write the KEY_IV_DATA_IN_CLEAR and DATA_OUT_CLEAR trigger bits to 1 and wait for it to
+              complete by polling the status idle bit.
+            - Read back the data out CSRs - they should all read garbage values.
+            - Assertion check verifies that the IV are also garbage, i.e. different from the
+              originally written values.
+            '''
+      features: [
+        "AES.MODE.ECB",
+        "AES.CLEAR.DATA_OUT",
+        "AES.CLEAR.KEY_IV_DATA_IN",
+      ]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["PROD"]
+      tests: ["chip_sw_aes_entropy"]
+    }
+    {
+      name: chip_sw_aes_prng_reseed
+      desc: '''Verify that prng reseeding is triggered
+
+            This test should start an encryption of a long message, disable the EDN complex, and
+            verify that the the encryption stalls when reseed is required.
+
+            Procedure:
+            -Configure encryption using the desired mode, key length, IV, and set the engine to
+            automatic operation.
+                -Set the mask_reseeding to once per 64 blocks (kDifAesReseedPer64Block)
+            -Start encryption of a long message (longer than 64 blocks)
+            -Disable the EDN complex using dif_edn_stop().
+            -Verify that the encryption stalls after 64 blocks are encrypted.
+
+            -Repeat the test for a reseed rate kDifAesReseedPer8kBlock and a message longer than 8K
+            blocks.
+            '''
+      features: [
+        "AES.MODE.ECB",
+        "AES.PRNG.RESEED_RATE",
+      ]
+      stage: V2
+      si_stage: SV3
+      lc_states: ["PROD"]
+      tests: ["chip_sw_aes_prng_reseed"]
+    }
+    {
+      name: chip_sw_aes_force_prng_reseed
+      desc: '''Verify that when KEY_TOUCH_FORCES_RESEED is set, the prng reseed is
+               triggered every time a key changes
+
+            Procedure:
+            -Configures the auxiliary options for AES to enable KEY_TOUCH_FORCES_RESEED
+            -Disable the EDN complex using dif_edn_stop()
+            -Configure the encryption, mode, key length, and set the engine to automatic
+             operation.
+            -Verify that the encryption stalls.
```

Comment:
Same here: re-enable the entropy complex to ensure after AES gets the entropy, it can indeed advance.